types/styled-system/index.d.ts

```diff
@@ -7,6 +7,7 @@
 //                 Eloy Durán <https://github.com/alloy>
 //                 Matthieu Vachon <https://github.com/maoueh>
 //                 Adam Lavin <https://github.com/lavoaster>
+//                 Nicholas Pappagalo <https://github.com/nicholaai>
```

Comment:
Offtopic: just want to make sure you’re aware that this means you’re signing up to perform future PR reviews?

I’ll leave actual review of this PR to others that more actively use this package.